### DISCUSSION TITLE
remote repo is different to local repo after pushing everything

### DISCUSSION THREAD

**Discussion by jonatsuji** [neuefische/questions#406] [Web] [answered]
Hey again,

yesterday I pushed my feature branch "feature/implement-area-cards". In there, I had the two folders "sectors" and "routes" written with lower first-letter. In my remote repo on the same branch, even after the second push, the folders are written with upper first-letter. This causes that my routing doesn't work anymore and now I don't know how to get my local repo (correct version) to overwrite the remote repo. Another git push didn't help.
Here's my current [PR](https://github.com/jonatsuji/capstone-project/pull/2/files) with the wrong folder names.

![Screenshot 2022-12-16 at 11 22 42](https://user-images.githubusercontent.com/115465230/208077787-5914345d-586c-44a7-9a05-d0ebbcc67f69.png)
![Screenshot 2022-12-16 at 11 23 37](https://user-images.githubusercontent.com/115465230/208077794-7ac0a042-3511-44ba-a5bc-9a576299a865.png)

Many thanks.

**Comment by MerleMak** [comment, marked as answer]
Hi Jonathan, this is an annoying feature of Mac. If you only change folder or filenames and nothing else and try to push, git won't recognise it. You have to change somethings in your code. Even if it is only a comment or a Space. 
Unfortunately this causes some trouble sometimes. Try again with a change in a file and let me know if it works :)

**Reply by jonatsuji** [reply]
Hi @MerleMak , 
Now, I changed a part of code, but also my last push was already with code changes and the changes I make on the code are shown in the remote repo but the folder just stay the same :( so no improve here...

**Comment by stefankorder** [comment]
You could also try this: https://github.com/orgs/neuefische/discussions/347?#discussioncomment-4274683
